python/cudf/utils/ioutils.py

```diff
@@ -1,18 +1,23 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
 from cudf.utils.docutils import docfmt_partial
 
 _docstring_read_parquet = """
-Load a parquet object from the file path, returning a DataFrame.
+Read a Parquet file into DataFrame
 
 Parameters
 ----------
-path : string
-    File path
+path : string or path object
+    Path of file to be read
+engine : { 'pyarrow', 'cudf' }, optional
```

Comment:
Would you be comfortable making `cudf` the default engine used? If not I'd want to keep the warning about going through the CPU for `pyarrow` and add a warning for `cudf` that the GPU parquet reader is experimental.

Reply:
Yeah, I've defaulted to `cudf` backend now. We've tested with several Parquet example and test files and it all looks okay. It's pretty much feature complete in the files we've encountered, except for less common compression codecs.

The good thing is that unlike CSV, there aren't a huge list of configurable parameters that we needed to handle.